content/SCALE/SCALEUIReference/Shares/iSCSISharesScreens.md

```diff
@@ -11,6 +11,7 @@ tags:
 - iscsi
 - shares
 - iscsi
+- high availability (ha)
```

Comment:
HA systems can use iSCSI shares but so do single controller non-HA systems. Remove this mention as it is not really a correct usage.
```suggestion
```